AK/BitmapView.h

```diff
@@ -64,19 +64,19 @@ class BitmapView {
                 count += popcount(byte);
             }
             if (++first < last) {
-                size_t const* ptr_large = (size_t const*)(((FlatPtr)first + sizeof(size_t) - 1) & ~(sizeof(size_t) - 1));
-                if ((u8 const*)ptr_large > last)
-                    ptr_large = (size_t const*)last;
-                while (first < (u8 const*)ptr_large) {
+                size_t const* ptr_large = reinterpret_cast<size_t const*>((reinterpret_cast<FlatPtr>(first) + sizeof(size_t) - 1) & ~(sizeof(size_t) - 1));
+                if (reinterpret_cast<u8 const*>(ptr_large) > last)
+                    ptr_large = reinterpret_cast<size_t const*>(last);
+                while (first < reinterpret_cast<u8 const*>(ptr_large)) {
```

Comment:
These and all pointer casts in BitmapView can be bit_cast, since that adds an extra layer of size and type checking (you can accidentally reinterpret-cast many things that are not of the same size, while this is not possible with bit-cast except when padding is involved).